docs/getting-started/integrate-auth/14_auth-js.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: auth-js
+title: Integrate authentication into Auth.js and NextAuth
+sidebar_label: Auth.js and NextAuth
+---
+
+This guide covers integrating Ory with Auth.js, a popular authentication library for Next.js applications. Auth.js is a flexible
+authentication library that supports multiple authentication providers, including Ory Network out of the box.
+
+Auth.js is the successor to NextAuth.js, which is a popular authentication library for Next.js applications. The Ory provider
+works with both Auth.js and NextAuth.js, so you can use either library to integrate Ory with your Next.js application.
+
+To connect your Next.js application with Ory we:
+
+1. Clone an example Next.js application with Auth.js
+2. Create an OAuth2 client in Ory and configure Auth.js to use it
+3. Perform a demo flow to see the integration in action
+
+Let's get started!
+
+## Clone the example Next.js application
+
+First, clone the example Next.js application with Auth.js:
+
+```shell-session
+git clone https://github.com/nextauthjs/next-auth-example.git
+cd next-auth-example
+npm i
+cp .env.local.example .env.local
+npx auth secret
+```
+
+In the `auth.ts` file, replace the `profiders` array with just Ory:
+
+```ts title="auth.ts"
+import { OryNetworkCta } from './ory-network-cta'
+export const config = {
+  theme: { logo: "https://authjs.dev/img/logo-sm.png" },
+  providers: [
+    // Apple,
+    // AzureB2C,
+    Ory
+  ]
+  // ...
+}
+```
+
+## Create OAuth2 Client in Ory Network
+
+To create the OAuth2 client, you need to know the redirect URL of your Next.js application. The redirect URL is the URL where Ory
+will send the user after they log in. When running NextJS locally, the redirect URL for Auth.js is usually
+`http://localhost:3000/auth/callback/ory`.
+
+1. Log into your Ory Network account.
+2. create a new project, or select an existing one.
+3. Go to ["OAuth 2" -> "Clients"](https://console.ory.sh/projects/current/oauth) and click
+   ["Create OAuth 2.0 Client"](https://console.ory.sh/projects/current/oauth/create).
+4. Select "Server App".
+5. Choose a client name, e.g. "NextAuth Example".
+6. Scopes `openid` and `offline_access` are preselected. Add `email` and `profile` to the list.
+7. Add the redirect URL, for example `http://localhost:3000/api/auth/callback/ory`.
```

Comment:
```suggestion
7. Add the redirect URL, for example, `http://localhost:3000/api/auth/callback/ory`.
```

Reply:
What's strange, I had to use `http://localhost:3000/auth/callback/ory` as the Callback URL. Maybe it's because of the new app router? I'm not sure. Probably should just add both of the URLs to the allowed redirect URLs.